docs/specification.md

```diff
@@ -29,6 +29,10 @@ These two parts are coupled into one smart contract because interaction between
 
```

Comment:
A few lines above, in the overview there is:
```
These two parts are coupled into one smart contract because interaction between smart contracts in Tezos is expensive and hard to get right.
```
I think we should change it to:
```
These two parts are coupled into one smart contract to reduce interactions between smart contracts, which is expensive and hard to get right in Tezos.
However, an existing separate FA2 contract is used for governance.
```

Reply:
Have changed.